src/js/client/main.js

```diff
@@ -31,15 +31,15 @@ $(document).ready(function () {
         .slideDown();
     };
 
-    $this.submit(function (e) {
+    $this.submit((e) => {
       e.preventDefault();
 
       $this.find('.subscribe-form-output')
         .slideUp(400, function () {
           $(this).remove();
         });
 
-      if ($this.find('.subscribe-form-email').val().trim() === "") {
+      if ($this.find('.subscribe-form-email').val().trim() === '') {
```

Comment:
Is using single quotes a js-lint thing for consistency?

Reply:
It is yep